frontend/src/components/contact/hooks/useUpdateContact.ts

```diff
@@ -0,0 +1,47 @@
+import axios, { AxiosError } from 'axios';
+import { useApiContacts } from 'hooks/pims-api/useApiContacts';
+import { IEditablePerson } from 'interfaces/editable-contact';
+import { IApiError } from 'interfaces/IApiError';
+import { useDispatch } from 'react-redux';
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import { toast } from 'react-toastify';
+import { logError } from 'store/slices/network/networkSlice';
+
+/**
+ * hook that updates a contact.
+ */
+export const useUpdateContact = () => {
+  const { putPerson } = useApiContacts();
+  const dispatch = useDispatch();
+
+  const updatePerson = async (person: IEditablePerson) => {
```

Comment:
same question, this component doesn't allow existing contacts to be edited does it? It just allows them to select one.